app/templates/components/n-times.hbs

```diff
@@ -0,0 +1,3 @@
+{{#each (range 1 (inc times)) as |number|}}
```

Comment:
Instead of incrementing the number, you could start the range from 0 right ? One less operation.